ttnn/cpp/ttnn/operations/experimental/reduction/argmax/argmax.cpp

```diff
@@ -131,9 +131,10 @@ Tensor _argmax(const Tensor& input_t, int64_t _dim, bool all, const std::optiona
 
 }
 
-Tensor _argmin(const Tensor& input_a, int64_t _dim, bool all, const std::optional<MemoryConfig>& output_mem_config) {
-    Tensor neg_input = ttnn::neg(input_a, output_mem_config);
-    return _argmax(neg_input, _dim, all, output_mem_config);
+Tensor ArgminOperation::operator()(const Tensor& input_a, int64_t _dim, bool all, const std::optional<MemoryConfig>& output_mem_config) {
+    auto output_memory_config = output_mem_config.value_or(input_a.memory_config());
+    Tensor neg_input = ttnn::neg(input_a, output_memory_config);
+    return ArgmaxOperation::operator()(neg_input, _dim, all, output_memory_config);
```

Comment:
should we use ttnn::argmax() instead ?

Reply:
Yes, definitely should use ttnn::argmax instead. 
ttnn::argmax is a registered, decorated version of the call and we should use it